guides/source/index.html.erb

```diff
@@ -7,6 +7,10 @@
 
 <% content_for :index_section do %>
 <nav id="subCol" aria-label="Chapter" class="guide-index">
```

Comment:
Yeah I agree. It believe the choice was made to have it as nav because on articles it does contain links but on the index it doesn't.

Let's change it here.